libraries/botbuilder-dialogs-adaptive/src/entityAssignmentComparer.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { AdaptiveEvents } from './adaptiveEvents';
+import { EntityAssignment } from './entityAssignment';
+
+export class EntityAssignmentComparer {
+    private static eventPreference: string[] = [
+        AdaptiveEvents.assignEntity,
+        AdaptiveEvents.chooseProperty,
+        AdaptiveEvents.chooseEntity,
+    ];
+
+    constructor(private operationPreference: string[]) {}
```

Comment:
This value seems unused?

Reply:
That's only because I ported it over poorly :(